client/app/scripts/superdesk-archive/views/metadata-view.html

```diff
@@ -17,9 +17,9 @@
             <dt translate>Word Count</dt>
             <dd>{{ item.word_count }}</dd>
         </dl>
-        <dl>
+        <dl ng-if="item.keywords && item.keywords.length > 0">
```

Comment:
fyi you can use just `item.keywords.length > 0`, angular will handle cases when keywords is undefined etc.